test/uk/gov/hmrc/mtdsatestsupportapi/models/request/createTestBusiness/QuarterlyTypeChoiceSpec.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mtdsatestsupportapi.models.request.createTestBusiness
+
+import api.models.domain.TaxYear
+import play.api.libs.json.Json
+import support.UnitSpec
+
+class QuarterlyTypeChoiceSpec extends UnitSpec {
```

Comment:
Should probably be testing both reads and writes. See [here](https://github.com/hmrc/business-details-api/blob/main/test/v1/models/response/retrieveBusinessDetails/downstream/QuarterTypeElectionSpec.scala)